docs/source/getting_started/index.rst

```diff
@@ -1,99 +1,123 @@
 Getting started
 ===============
 
-In this section, we will create an environment to run Iroha instance, play around with settings a bit, launch it, create a couple of transactions, and check the data written in the ledger. Also, we will deploy a network of peers and will try to emulate rainy day scenarios happening in the network.
+In this guide, we will create a very basic Iroha network, launch it, create a couple of transactions, and check the data written in the ledger. To keep things simple, we will use Docker.
 
 .. note:: Ledger is the synonym for a blockchain, and Hyperledger Iroha is known also as Distributed Ledger Technology — which in essence is the same as "blockchain framework". You can check the rest of terminology used in the Glossary section.
 
-Let's prepare the environment
------------------------------
+Prerequisites
+-------------
+For this guide, you need a computer running  Unix-like system with ``docker`` installed. You can read how to install it on a `Docker's website <https://www.docker.com/community-edition/>`_
 
-.. hint:: Having troubles? Check FAQ section or communicate to us directly, in case you were stuck on something. We don't expect this to happen, but some issues with an environment are possible.
+.. note:: Please note that you can use Iroha without ``docker`` as well. You can read about it in other parts of documentation.
 
-Prerequisite 1
-^^^^^^^^^^^^^^
+Starting Iroha Network
+----------------------
 
-Prerequisite 2
-^^^^^^^^^^^^^^
+Creating a Docker network
+^^^^^^^^^^^^^^^^^^^^^^^^^
+To operate, Iroha requires a ``PostgreSQL`` database. Let's start with creating Docker network, so containers for Postgres and Iroha can run in the same virtual network and successfully communicate. In this guide we will call it ``iroha-network``, but you can use any name. In your terminal write following command:
 
-Prerequisite 3
-^^^^^^^^^^^^^^
+.. code-block:: shell
 
-Getting Iroha build
--------------------
+  docker network create iroha-network
 
+Starting PostgreSQL container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Step 1
-^^^^^^
+Now we need to run ``PostgreSQL`` in a container, attach it to the network you have created before, and exposing ports for communication:
 
-Step 2
-^^^^^^
+.. code-block:: shell
 
-Step 3
-^^^^^^
+  docker run --name some-postgres \
+  -e POSTGRES_USER=postgres \
+  -e POSTGRES_PASSWORD=mysecretpassword \
+  -p 5432:5432 \
+  --network=iroha-network \
+  -d postgres:9.5
 
-Playing around with settings
-----------------------------
+Creating blockstore
+^^^^^^^^^^^^^^^^^^^
+Before we run Iroha container, we should create persistent volume to store files, storing blocks for the chain. It is done via the following command:
 
+.. code-block:: shell
 
-Step 1
-^^^^^^
+  docker volume create blockstore
 
-Step 2
-^^^^^^
+Configuring Iroha network
```

Comment:
Please tell that you are actually going to run a single peer, and put a link to more advanced tutorial with several peers running in a network